### PR TITLE
Andres delegate deep link

### PR DESCRIPTION
**Pull request by andresinaka**
🪄 
https://user-images.githubusercontent.com/1957233/142875076-e7855efd-ed74-4eec-a1ba-1c492a266ffa.mp4

### CODE REVIEW

File: ProxyRelease/ProxyReleaseApp.swift [hunk 1]
```diff
@@ -7,11 +7,48 @@
...
+    init() {
+        print("Here is where we are going to init things. Equivalent to didFinishLaunchingWithOptions")
+    }
+
     var body: some Scene {
         WindowGroup {
             RootView(viewModel: RootViewModel())
+                .onOpenURL { url in // Handling deep links links
+                    print("Handled in ProxyReleaseApp: \(url)")
+                }
```
Comment (facumenzella) [inline ProxyRelease/ProxyReleaseApp.swift:40]: I've just read about this. Could we define this multiple times?

Comment (andresinaka) [reply ProxyRelease/ProxyReleaseApp.swift:40]: yes! we can have it multiple times

File: ProxyRelease/RootView.swift [hunk 2]
```diff
@@ -35,15 +36,21 @@ struct RootView: View {
...
 
                 PendingTasksView()
                     .tabItem {
                         Image(systemName: "list.bullet.rectangle")
                         Text(RootViewModel.Tab.tasks.navTitle)
                     }
                     .tag(RootViewModel.Tab.tasks)
+                    .navigationBarTitle(viewModel.selectedTab.navTitle)
             }
-            .navigationBarTitle(viewModel.selectedTab.navTitle)
```
Comment (raluca-toadere-prx) [inline ProxyRelease/RootView.swift:46]: Why do we need to move back the navigationBarTitle modifier to each of the tab views?

Comment (andresinaka) [reply ProxyRelease/RootView.swift:46]: not needed! I picked the wrong side of the merge conflict.

**Review by raluca-toadere-prx — APPROVED** [review]

**Review by facumenzella — APPROVED** [review]

**Pull request merged**